Guide.docc/IncrementalAdoption.md

```diff
@@ -227,3 +227,98 @@ NS_SWIFT_ASYNC_NAME
 NS_SWIFT_ASYNC_NOTHROW
 NS_SWIFT_UNAVAILABLE_FROM_ASYNC(msg)
 ```
+
+### Dealing mistakes in isolation annotations in Objective-C libraries
+
+While the SDKs and other objective-c libraries make progress in adopting Swift concurrency,
+they will often go through the exercise of codifying contracts which were only explained in
+documentation, like "this will always be called on the main thread", and turn them intro 
+compile and runtime enforced isolation checks that Swift will then perform when you adopt such APIs.
```

Comment:
done thank you!

> While the SDKs and other Objective-C libraries make progress in adopting Swift concurrency,
they will often go through the exercise of codifying contracts which were only explained in
documentation. For example, before Swift concurrency, APIs frequently had to document their
threading behavior with comments like "this will always be called on the main thread".
> 
> Swift concurrency enables us to turn these code comments, into compiler and runtime 
enforced isolation checks, that Swift will then verify when you adopt such APIs.